src/apps/common/submodules/portListing/portListing.js

```diff
@@ -552,21 +554,21 @@ define(function(require) {
 					}));
 				});
 
-			template
+			textareaWrapper
 				.find('#add_comment')
 					.on('click', function(event) {
-						event.preventDefault();
-
-						var textarea = template.find('textarea[name="comment"]'),
-							comment = textarea.val();
+						self.portListingHelperPostComment(event, textarea, args, false);
```

Comment:
Looks good,

Please consider to send parameters like an object, also I think you don't need to send the `onClick event` to `portListingHelperPostComment`, also with that way should be pretty simple tu define a default value for `isSuperDuperComment`
```suggestion
						event.preventDefault();

						self.portListingHelperPostComment({
							textarea: textarea,
							args: args
						});
```